extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -627,9 +690,7 @@ def validate(self, classifier_data):
             '_w_dp',
             '_l_dp',
             '_c_dl',
-            '_c_lw'
-        ]
-        list_of_list_bool_properties = [
+            '_c_lw',
             '_b_dl'
```

Comment:
Seems a bit fishy. If you are trying to represent bools as 0/1 instead of False/True then you should still have validation to ensure that these are the only allowed values.